src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -94,5 +100,11 @@ const goToKeywordDensity = () => {
       </div>
     </div>
   </div>
-  <MaButton highlight class="flex-1 justify-center ml-1 mt-2" @click="goToKeywordDensity" >Go to Keyword Density Calculator'</MaButton>
 </template>
+
+<style>
+.ma-select {
+  width: 150px !important;
```

Comment:
we should not use `!important` unless we really need it. You can read more in this article: https://element.how/why-you-should-avoid-using-important-in-css/